airbyte-integrations/connectors/source-bigquery/source_bigquery/source.py

```diff
@@ -194,12 +193,19 @@ def _set_sync_mode(self):
                     if configured_stream.stream.name == stream.name and configured_stream.sync_mode:
                         stream.configured_sync_mode = configured_stream.sync_mode
 
-    def _get_tables(self, project_id, dataset_id, table_id, sync_method, slice_range, stream_name=None, where_clause=""):
+    def _get_tables(self, project_id, dataset_id, table_id, sync_method, slice_range, table_info, stream_name=None, where_clause=""):
         # list and process each table under each base to generate the JSON Schema
         if sync_method == "Standard":
             table_obj = IncrementalQueryResult(project_id, dataset_id, table_id, given_name=stream_name, where_clause=where_clause, fallback_start=FALLBACK_START, slice_range=slice_range, authenticator=self._auth)
         else:
-            table_obj = TableChangeHistory(project_id, dataset_id, table_id, given_name=stream_name, where_clause=where_clause, fallback_start=CHANGE_HISTORY_START, slice_range=slice_range, authenticator=self._auth)
+            table_creation_datetime = pendulum.from_timestamp(float(table_info["creationTime"])/1000.0) # timestamps returned are in milliseconds hence the /1000
```

Comment:
For Snowflake, we had the issue where the creationTime would be updated during `create or replace table` statement which was annoying because there were records before the newly updated creationTime. 

Example:
* T0: create table. creationTime would be `T0`
* T1: Add records
* T2: replace table using `create or replace` statement. creationTime would be `T2`

Can we confirm that this is not the case for BigQuery?

Reply:
Ok I will test this case